src/Lumi/Components/Modal.purs

```diff
@@ -124,7 +124,7 @@ type CommonProps rest =
   { modalOpen :: Boolean
   , onActionButtonClick :: Nullable (Effect Unit)
   , actionButtonTitle :: String
-  , actionButtonDisabled :: Boolean
+  , actionButtonDisabled :: Button.ButtonState
```

Comment:
Since this is a breaking change anyway, should we rename this prop to `actionButtonState`?

Reply:
Good call will update.